apps/schematic/api/Dockerfile

```diff
@@ -13,21 +13,21 @@ ENV PYTHONFAULTHANDLER=1 \
     PIP_DISABLE_PIP_VERSION_CHECK=on \
     PIP_DEFAULT_TIMEOUT=200 \
     POETRY_VERSION=1.3.0 \
-    APP_PARENT_DIR=/app \ 
+    APP_PARENT_DIR=/app \
     NGINX_CONFIG=/etc/nginx/conf.d \
     APP_DIR=/app/app \
     ROOT=/ \
-    UWSGI_INI=/app/uwsgi.ini \ 
+    UWSGI_INI=/app/uwsgi.ini \
     NGINX_WORKER_PROCESSES=1 \
     VERSION=$TAG
 
 # run open ssl and generate certificate
-RUN apt update \
+RUN apt-get update -qq -y && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get install --no-install-recommends -qq -y \
+      openssl jq \
     && apt-get -y autoclean \
     && apt-get -y autoremove \
     && rm -rf /var/lib/apt/lists/* \
```

Comment:
shouldn't that `\` be removed? there's nothing after `\`

Reply:
You're right. Removing it now...

Reply:
Done